apps/website/src/pages/docs/_layout.astro

```diff
@@ -16,6 +16,8 @@ const currentPage = new URL(Astro.request.url).pathname;
     <slot />
     <div class='bottom-links'>
       <hr />
+      <h3>Help us to improve our documentation</h3>
+      <p>Suggest an edit to this page, or provide / view feedback for this page.</p>
       <EditPageLink page={currentPage} />
```

Comment:
"Edit this page" and "provide feedback" should be separate components. And the surrounding text is not really needed as the links are already pretty clear.